var/spack/repos/builtin/packages/opari2/package.py

```diff
@@ -39,14 +39,12 @@ class Opari2(Package):
     homepage = "http://www.vi-hps.org/projects/score-p"
     url      = "http://www.vi-hps.org/upload/packages/opari2/opari2-1.1.2.tar.gz"
 
-    version('2.0', '72350dbdb6139f2e68a5055a4f0ba16c',
-            url='http://www.vi-hps.org/upload/packages/opari2/opari2-2.0.tar.gz')
-    version('1.1.4', '245d3d11147a06de77909b0805f530c0',
-            url='http://www.vi-hps.org/upload/packages/opari2/opari2-1.1.4.tar.gz')
+    version('2.0.1', '74af78f1f27b8caaa4271e0b97fb0fba')
+    version('2.0',   '72350dbdb6139f2e68a5055a4f0ba16c')
+    version('1.1.4', '245d3d11147a06de77909b0805f530c0')
     version('1.1.2', '9a262c7ca05ff0ab5f7775ae96f3539e')
 
-    def install(self, spec, prefix):
-        configure("--prefix=%s" % prefix,
-                  "--enable-shared")
-        make()
-        make("install")
+    def configure_args(self):
+        return ["--enable-shared"]
+
+# -----------------------------------------------------------------------------
```

Comment:
I don't think we want this comment at the end. At least, no other package in Spack has one.